google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/BlobInfo.java

```diff
@@ -260,11 +260,14 @@ static CustomerEncryption fromPb(StorageObject.CustomerEncryption customerEncryp
     abstract Builder setKmsKeyName(String kmsKeyName);
 
     /** Sets the blob's event-based hold. */
+    @GcpLaunchStage.Beta
```

Comment:
I think you're looking for the `com.google.api.core.BetaApi` annotation.

Reply:
Updated annotation. Thanks @chingor13